client/renderSDL/SDLImage.h

```diff
@@ -66,6 +67,20 @@ class SDLImageShared final : public ISharedImage, public std::enable_shared_from
 	friend class SDLImageLoader;
 };
 
+class SDLEmptyImageShared : public SDLImageShared
+{
+public:
+	SDLEmptyImageShared() : SDLImageShared(createDefaultSurface().get()) {}
+
+private:
+	std::shared_ptr<SDL_Surface> createDefaultSurface()
+	{
+		auto surface = SDL_CreateRGBSurface(0, 32, 32, 32, 0, 0, 0, 0);
+		SDL_FillRect(surface, nullptr, SDL_MapRGB(surface->format, 255, 0, 0));
+		return std::shared_ptr<SDL_Surface>(surface, SDL_FreeSurface);
+	}
+};
+
```

Comment:
Unused? 
Also - what's exact point of this class?

If you're implementing my TODO, then its goal was somewhat different - it was to create dummy implementation of ISharedImage to replace checks for non-null surface in every method of SDLImageShared

Reply:
Just tried to implement some "default" as I didn't realise that there is sth like that already.